src/units/attack_type.cpp

```diff
@@ -164,6 +165,7 @@ static bool matches_simple_filter(const attack_type & attack, const config & fil
 
 
 	if(!filter_special_active.empty()) {
+		deprecated_message("special_active=", DEP_LEVEL::PREEMPTIVE, {1, 16, 0}, "Please use special_id_active or special_type_active instead");
```

Comment:
Looks good, but the version should be `{1, 17, 0}`.

Reply:
it is done

Reply:
You need to include deprecation.hpp and game_version.hpp to fix a build error.